action.yml

```diff
@@ -146,6 +140,7 @@ runs:
       shell: bash
       env:
         BRANCHES: ${{ steps.determine.outputs.branches }}
+        GITHUB_REPOSITORY: ${{ github.repository }}
```

Comment:
This env var is being added for the github api request to trigger a CI  build - it should have been there - possibly this wasn't pushed last time.